Makefile

```diff
@@ -0,0 +1,31 @@
+MSBUILD = xbuild
+MDOC = mdoc/bin/Release/mdoc.exe
+
+all: build
+
+build: $(MDOC)
+$(MDOC): external/SharpZipLib/bin/Release/ICSharpCode.SharpZipLib.dll external/Mono.Cecil/bin/net_4_0_Release/Mono.Cecil.dll
+	$(MSBUILD) apidoctools.sln /p:Configuration=Release
+
+prepare:
+	git submodule update --init --recursive
+	nuget restore -PackagesDirectory Packages
```

Comment:
Why `Packages` and not `packages`? The typical directory appears to be `packages`, based on the number of them I have from `nuget restore`...

Differences in casing can cause issues on case-sensitive filesystems. We should be consistent.